Earthfile

```diff
@@ -63,6 +66,7 @@ prepare-cache:
     COPY --keep-ts crates/dbsp/Cargo.toml crates/dbsp/
     COPY --keep-ts crates/adapters/Cargo.toml crates/adapters/
     COPY --keep-ts crates/pipeline_manager/Cargo.toml crates/pipeline_manager/
+    COPY --keep-ts web-ui web-ui
```

Comment:
I think this will run install-chef every time the web-ui changes. Not sure it's needed? The build.rs file should be empty during the chef run?